meta-aos-rcar-gen4-dom0/recipes-core/runx/runx_git.bb

```diff
@@ -0,0 +1,60 @@
+HOMEPAGE = "https://github.com/lf-edge/runx"
+SUMMARY = "runx stuff"
+DESCRIPTION = "Xen Runtime for OCI"
+
+SRCREV_runx = "0c7edb3453398d7a0c594ce026c9c1e93c2541cc"
```

Comment:
why do you need `_runx` append?

Reply:
This recipe was originally moved from meta-virtualization/recipes-core/runx/runx_git.bb. I've decided to override whole recipe because there are too huge changes in compare with the original one. For example - I've swithced to the branch, which doesn't use go to build sendfd program. Also, I've removed initrd and kernel build as not relevant for us.
As for the _runx it's also more precise way to set variables, by adding ${PN} at the end. It mostly needed when you build >1 packages in one recipe. I've decised to leave it as is because it's always good to be more precise.

Reply:
Now I understand where this recipe comes from. It would be nice to mention it in commit message form where the original recipe was taken. According to your description, there are not so many changes. So, having bbappend is preferable here.

Reply:
Ok. I'll make bbappend than.